_episodes/37.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: episode
+title:  "35: Packaging and modernizing Trino"
```

Comment:
37: Trino community support

Reply:
I want the title to be a little playful. How about:

> 37: Trino powers up the community support